documentation/release-notes/topics/version_4.md

```diff
@@ -1,5 +1,92 @@
 # Version 4
 
+## Hyperview 4.8 (TBD)
+
+This section covers significant changes and bug fixes in Hyperview 4.8.x since version 4.7.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating your installed Data
+   Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+2. Support for the *Windows* version of the Data Collector has ended on *January 31, 2025*. Customers must switch
+   to the Linux version of the Data Collector. Detailed {ref}`installation instructions <Setting-up-data-collectors-doc>`
+   are in the product documentation.
+
+3. The communication provider for email notification has been changed from Twilio SendGrid to Microsoft Azure Communication
```

Comment:
production emails will continue to come from no-reply@hyperviewhq.com, the azurecomm domains are just for development

Reply:
Thanks! I added a note about maintaining the From address.